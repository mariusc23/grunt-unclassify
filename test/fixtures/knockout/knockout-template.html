<script>alert('hi');</script>
<script src="script.js"></script>
<noscript>noscript.js</noscript>

<script id="testTemplate" type="text/html">
  <div class="only"></div>
  <div class="exists-only"></div>
</script>
