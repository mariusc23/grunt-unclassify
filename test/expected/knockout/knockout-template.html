<script>alert('hi');</script>
<script src="script.js"></script>
<noscript>noscript.js</noscript>

<script id="testTemplate" type="text/html">
  <div></div>
  <div class="exists-only"></div>
</script>
